src/main/java/com/red5pro/cluster/streams/Provision.java

```diff
@@ -0,0 +1,69 @@
+package com.red5pro.cluster.streams;
+
+import java.util.List;
+
+/**
+ * 
+ * @author Andy Shaules
+ *
+ */
+public class Provision {
+	/**
+	 * Returns concatenated context path.
+	 * @param context app scope
+	 * @param name stream name
+	 * @return String concatenated with "/" between context and name
+	 */
+	public static String GetGuid(String context, String name){
+		if(context.startsWith("/")){
```

Comment:
This has bad camel casing; I'd suggest `findGuid()` instead since `getGuid()` exists below.

Reply:
bad naming. should be MakeGuid